builder-image-gitops/go/src/gitops/main_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+  "testing"
+  "github.com/hashicorp/hcl"
+)
+
+func TestDecodeHcl(t *testing.T) {
+
+  hclString := "inputs = {\n  one = \"bar\"\n}"
+  ast, err := hcl.Parse(hclString)
+
+  if err != nil {
+    t.Error(err)
+    t.Log(ast)
+  }
+}
+
+func TestSetValueInAst(t *testing.T) {
```

Comment:
would be good to add tests for situations like referencing a nested object that doesn't exist ... `inputs.missing.two=foo`